### PR TITLE
[WIP] ENH Add BlazingSQL meta packages

### PR DESCRIPTION
**Pull request by dillon-cullinan**
* Adds `blazingsql-build-env` meta package
* Adds `blazingsql-notebook-env` meta package
* New `rapids-blazing` package now contains `blazingsql`

**Note**: Building the notebook environment requires two additional channels. `plotly` and `pytorch`

`rapids-blazing`

```
$ conda search -i -c /home/nfs/dcullinan/miniconda3/conda-bld rapids-blazing

Loading channels: done
rapids-blazing 0.17.0a cuda10.2_py37_gb949b3f_110
-------------------------------------------------
file name   : rapids-blazing-0.17.0a-cuda10.2_py37_gb949b3f_110.tar.bz2
name        : rapids-blazing
version     : 0.17.0a
build       : cuda10.2_py37_gb949b3f_110
build number: 110
size        : 4 KB
license     : Custom
subdir      : linux-64
url         : file:///home/nfs/dcullinan/miniconda3/conda-bld/linux-64/rapids-blazing-0.17.0a-cuda10.2_py37_gb949b3f_110.tar.bz2
md5         : 96d2add13b0e6ee8248346c165638505
timestamp   : 2020-11-24 18:08:01 UTC
dependencies:
  - blazingsql 0.17.*
  - cudatoolkit 10.2.*
  - python_abi 3.7.* *_cp37m
  - rapids 0.17.*
```

`blazingsql-build-env`

```
$ conda search -i -c /home/nfs/dcullinan/miniconda3/conda-bld blazingsql-build-env
Loading channels: done
blazingsql-build-env 0.0.0 cuda10.2_py37_g5aced79_102
-----------------------------------------------------
file name   : blazingsql-build-env-0.0.0-cuda10.2_py37_g5aced79_102.tar.bz2
name        : blazingsql-build-env
version     : 0.0.0
build       : cuda10.2_py37_g5aced79_102
build number: 102
size        : 4 KB
license     : Apache-2.0
subdir      : linux-64
url         : file:///home/nfs/dcullinan/miniconda3/conda-bld/linux-64/blazingsql-build-env-0.0.0-cuda10.2_py37_g5aced79_102.tar.bz2
md5         : 5e2cc18407afb49db7a2fcc96edf5da9
timestamp   : 2020-11-16 18:39:36 UTC
dependencies:
  - aws-sdk-cpp
  - cmake 3.18.*
  - cppzmq
  - cudatoolkit 10.2.*
  - cython >=0.29.17,<0.30
  - dlpack >=0.3,<0.4.0a0
  - google-cloud-cpp 1.16.0.*
  - gtest 1.10.0.*
  - jpype1
  - maven
  - netifaces
  - ninja
  - openjdk >=8.0
  - pyhive
  - python >=3.7,<3.8.0a0
  - python_abi 3.7.* *_cp37m
  - setuptools
  - spdlog 1.7.0.*
  - zeromq
```

`blazingsql-notebook-env`

```
$ conda search -i -c /home/nfs/dcullinan/miniconda3/conda-bld blazingsql-notebook-env
Loading channels: done
blazingsql-notebook-env 0.17.0a cuda10.2_py37_geb26574_111
----------------------------------------------------------
file name   : blazingsql-notebook-env-0.17.0a-cuda10.2_py37_geb26574_111.tar.bz2
name        : blazingsql-notebook-env
version     : 0.17.0a
build       : cuda10.2_py37_geb26574_111
build number: 111
size        : 4 KB
license     : Apache-2.0
subdir      : linux-64
url         : file:///home/nfs/dcullinan/miniconda3/conda-bld/linux-64/blazingsql-notebook-env-0.17.0a-cuda10.2_py37_geb26574_111.tar.bz2
md5         : 8d88a2e649eec9a7fad73d9435a27333
timestamp   : 2020-11-24 18:33:59 UTC
dependencies:
  - colorcet
  - cudatoolkit 10.2.*
  - dash
  - dask
  - dask-ml
  - datashader
  - fbprophet
  - geoviews
  - holoviews
  - ipywidgets
  - jupyter-dash
  - matplotlib
  - networkx
  - panel >=0.9.7
  - plotly
  - python-graphviz
  - python_abi 3.7.* *_cp37m
  - pytorch >=1.6
  - s3fs >=0.5.1
  - scikit-learn
  - scipy
  - seaborn
  - spacy
  - statsmodels
  - xgboost
```

### CODE REVIEW

**Review by ajschmidt8 — APPROVED** [review]

**Comment by dillon-cullinan** [comment]
rerun tests

**Comment by dillon-cullinan** [comment]
rerun tests

**Comment by dillon-cullinan** [comment]
rerun tests

**Comment by dillon-cullinan** [comment]
rerun tests

File: ci/cpu/build.sh [hunk 1]
```diff
@@ -50,17 +53,18 @@ if [[ "$BUILD_MODE" = "branch" && "$SOURCE_BRANCH" = branch-* ]] ; then
...
-EOF
-  cat ${1}/conda_build_config.yaml
   # Build pkg
   gpuci_logger "Start conda build for '${1}'..."
-  gpuci_conda_retry build --override-channels -c ${CONDA_USERNAME:-rapidsai-nightly} -c nvidia -c conda-forge -c defaults \
-              --python=${PYTHON_VER} -m ${CONDA_CONFIG_FILE} ${1}
+  if [[ ${1} == ${CONDA_BLAZING_NOTEBOOK_RECIPE} ]]; then
+    gpuci_conda_retry build --override-channels -c blazingsql-nightly -c ${CONDA_USERNAME:-rapidsai-nightly} -c nvidia -c pytorch -c conda-forge -c defaults \
+                --python=${PYTHON_VER} -m ${CONDA_CONFIG_FILE} ${1}
+  elif [[ ${1} == *"BLAZING"* ]]; then
```
Comment (mike-wendt) [inline ci/cpu/build.sh:61]: I'd wrap these variables `${1}` with quotes `"${1}"` just to be safe

File: ci/cpu/build.sh [hunk 2]
```diff
@@ -50,17 +53,18 @@ if [[ "$BUILD_MODE" = "branch" && "$SOURCE_BRANCH" = branch-* ]] ; then
...
-  cat > ${1}/conda_build_config.yaml <<EOF
-cuda_compiler_version:
-- '$CUDA_VER'
-EOF
-  cat ${1}/conda_build_config.yaml
   # Build pkg
   gpuci_logger "Start conda build for '${1}'..."
-  gpuci_conda_retry build --override-channels -c ${CONDA_USERNAME:-rapidsai-nightly} -c nvidia -c conda-forge -c defaults \
-              --python=${PYTHON_VER} -m ${CONDA_CONFIG_FILE} ${1}
+  if [[ ${1} == ${CONDA_BLAZING_NOTEBOOK_RECIPE} ]]; then
```
Comment (mike-wendt) [inline ci/cpu/build.sh:58]: I'd wrap these variables `${1}` with quotes `"${1}"` just to be safe

**Review by mike-wendt — APPROVED** [review]

**Pull request merged**